system-x/services/azure/storage-blob/src/main/java/software/tnb/azure/storage/blob/validation/StorageBlobValidation.java

```diff
@@ -50,4 +57,30 @@ public List<BlobItem> getBlockBlobs(String blobContainer) {
         LOG.debug("Getting block blobs from blob container {}", blobContainer);
         return client.getBlobContainerClient(blobContainer).listBlobs().stream().collect(Collectors.toList());
     }
+
+    public String getSas(String containerName, AzureAccount account) {
```

Comment:
`createSasToken` may be a better name

Reply:
add the `AzureAccount` to the constructor of `StorageBlobValidation` and pass it  [here](https://github.com/tnb-software/TNB/blob/main/system-x/services/azure/storage-blob/src/main/java/software/tnb/azure/storage/blob/service/StorageBlob.java#L38)